### DISCUSSION TITLE
EditableProTable组件可以支持回车保存吗？👑 [需求]

### DISCUSSION THREAD

**Discussion by egene-huang** [ant-design/pro-components#1317] [Q&A]
### 🥰 需求描述

 1. 是否可以支持回车保存吗？
 2. 当是已保存状态时， 是否可以直接漏出删除按钮？
 3. 行输入元素限制， 比如是number input?  

### 🧐 解决方案
无

### 🚑 其他信息

这个组件整体上已经很好了， 简单易用， 通过api 我相信后面还会有较大改动吧  比如: onlyOneLineEditorAlertMessage onlyAddOneLineAlertMessage  
之前有一个单元格编辑  使用起来炒鸡复杂  而且效果还不是很好  这次感谢这个组件!希望越来越好!!

**Comment by chenshuai2144** [comment]
> 1.是否可以支持回车保存吗？

这个可以做，做起来也比较简单

> 当是已保存状态时， 是否可以直接漏出删除按钮？

这个是根据你在不在 dataSource 里实现的

> 行输入元素限制， 比如是number input?

这个需求再描述一下了
